docs/public/secure-registry.adoc

```diff
@@ -35,3 +35,30 @@ Please run `odo registry add <registry name> <registry URL> --token <token>` to
 - <registry name>: user-defined devfile registry name.
 - <registry URL>: the URL of GitHub repository that you create on step 1.
 - <token>: the personal access token that you create on step 2.
+
+== Steps for setting up a secure starter project when the secure starter project is hosted on a GitHub repository
+
```

Comment:
This line is looking complicated to read

may be we can use 
`Steps for setting up a secure starter project on a github repository`